src/Integration.Vsix.UnitTests/CFamily/CFamilyHelperTests.cs

```diff
@@ -159,63 +173,53 @@ public void FileConfig_Test2()
         public void GetEvalutedPropertyValue_NoException_ReturnsValue()
```

Comment:
Test name is incorrect, should be something like 'GetPotentiallyUnsupportedPropertyValue_PropertySupported_ReturnsValue'